src/test/java/com/wikia/webdriver/elements/mercury/old/JoinPageObject.java

```diff
@@ -30,23 +28,22 @@
 
   public JoinPageObject(WebDriver driver) {
     this.driver = driver;
-    this.urlBuilder = new UrlBuilder();
     this.wait = new Wait(driver);
 
     PageFactory.initElements(driver, this);
   }
 
   public JoinPageObject get() {
     String redirectParameter = "";
-
+    UrlBuilder urlBuilder = UrlBuilder.createUrlBuilder();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "urlBuilder" which hides the field declared at line 27. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AHiddenFieldCheck)